core/src/View/Kokkos_ViewLegacy.hpp

```diff
@@ -291,6 +291,23 @@ class View : public ViewTraits<DataType, Properties...> {
   using uniform_runtime_const_nomemspace_type =
       typename Impl::ViewUniformType<View, 0>::runtime_const_nomemspace_type;
 
+  using reference_type = typename map_type::reference_type;
+  using pointer_type   = typename map_type::pointer_type;
```

Comment:
Technically you don't need the identifier attributes
```suggestion
  using typename map_type::reference_type;
  using typename map_type::pointer_type;
```
I don;t really care if you actually change

Reply:
haha I did what you said, and now I undo it again, because that only works for base-classes ...